cli/githubutil.py

```diff
@@ -186,6 +205,36 @@ def release_and_prepare_next_dev_cycle(
         commit_message="Prepare next dev cycle " + next_version_dev
     )
 
+def post_to_slack(
```

Comment:
I do not think this belongs into a module 'githubutil'. 'slack.util' seems more appropriate to me (or just slack)